### PR TITLE
Newsletters: add 147 (2021-05-05)

### PR DESCRIPTION
**Pull request by harding**
- [x] Update topics @harding 
- [x] `BTCPay 1.1.0` @bitschmidty 
- [x] `Bitcoin Core #19160` multiprocess: Add basic spawn and IPC support @jnewbery
- [x] `Bitcoin Core #21009` Remove RewindBlockIndex logic @Xekyo 
- [x] `LND #5159` AMP support for SendPaymentV2 @dongcarl

### CODE REVIEW

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  for their BIP32 wallet seed to re-discover their peers using only
+  public information about the LN network.  Once the user found their
+  peers, they could request the peers close their mutual channels using
+  the [BOLT2][] data loss protection protocol (see [Newsletter
+  #31][news31 data_loss]).  The proposed method isn't perfect---users
+  should still create backups[^missing-peer] and replicate them across
+  independent systems---but Fournier's proposal provides additional
+  redundancy that would be especially useful for everyday users.
+
+    A couple weeks ago, Rusty Russell restarted the [thread][russell
```
Comment (jnewbery) [inline _posts/en/newsletters/2021-05-05-newsletter.md:58]: I know that "a couple weeks" can be used in American English, but I don't know whether it's considered informal or not (in British English we'd always use "a couple of weeks"). I see a mixture of "a couple {things}" and "a couple of {things}" in previous newsletters.

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    ecdh channels] after trying to [specify][russell ecdh spec] and
+    implement the idea.  After some additional mailing list discussion with Fournier
+    and a group [conversation][lndev deterministic] in the weekly LN
+    protocol development meeting, Russell indicated he was leaning
+    against the idea, [saying][russell backups] "I see encrypted backups
+    as a more-likely-to-be-implemented solution though.  Because they're
+    useful to send to places other than peers, and they can also contain
+    [HTLC][topic htlc] information if you want."  Being able to contain
+    [HTLC][topic htlc] information would allow settling payments that
```
Comment (jnewbery) [inline _posts/en/newsletters/2021-05-05-newsletter.md:67]: No need to link to [topic htlc] from both of these. Perhaps remove the link from the Russell quote.

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  [342][BIP342].  Also included is the ability to pay [bech32m][topic
+  bech32] addresses specified by [BIP350][], although bitcoins spent to
+  such addresses on mainnet will not be secure until activation of a
+  soft fork using such addresses, such as taproot.  The release
+  additionally includes bug fixes and minor improvements.
+
+    Note: due to a [problem][wincodesign] with the certificate authority
+    that provides the code signing certificates for the Windows versions
+    of Bitcoin Core, users on Windows will need to click through an
+    extra prompt to install.  It is also expected that there will be a
```
Comment (jnewbery) [inline _posts/en/newsletters/2021-05-05-newsletter.md:91]:
```suggestion
    extra prompt to install.  It is expected that there will be a
```

**Review by jnewbery — APPROVED** [review]

**Comment by jnewbery** [comment]
Ooops. Didn't mean to click on 'approve'. Not sure how I can remove that.

**Comment by jnewbery** [comment]
Added Bitcoin Core 19160

File: _posts/en/newsletters/2021-05-05-newsletter.md (hunk 2 above)
Comment (harding) [reply _posts/en/newsletters/2021-05-05-newsletter.md:67]: Whoops, thanks.  (I added the link after pasting in Rusty's quote, then wrote the subsequent line and thought the link would be better there, then forgot to remove the old link.)

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19160][] is the next PR in the [multiprocess project][bitcoin
+  core multiprocess] and adds the ability for the `bitoin-node` process to spawn
+  other processes and communicate with them using [Capn' Proto][capn proto].
```
Comment (jonatack) [inline _posts/en/newsletters/2021-05-05-newsletter.md:114]:
```suggestion
  other processes and communicate with them using [Cap’n Proto][capn proto].
```
(noticed while opening the file; haven't read the rest yet and will try to do a full read-through a bit later)

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+  These features are currently only used for testing, but the [next PR][Bitcoin
+  Core #10102] in the project will allow Bitcoin Core to be started in
+  multiprocess mode with the `bitcoin-core` process spawning separate
+  `bitcoin-wallet` and `bitcoin-gui` processes.
+
+- [Bitcoin Core #21009][] removes the RewindBlockIndex logic triggered when
+  updating a pre-segwit client (v0.13.0 or older) to a segwit-enforcing
```
Comment (harding) [inline _posts/en/newsletters/2021-05-05-newsletter.md:121]: @Xekyo FYI, I'm going to s/client/node/ throughout this paragraph.  I prefer to use "client" for tools that are dependent on other tools, e.g. lightweight clients.  I use "node" or "peer" for tools that are full participants in the network.

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19160][] is the next PR in the [multiprocess project][bitcoin
+  core multiprocess] and adds the ability for the `bitoin-node` process to spawn
```
Comment (adamjonas) [inline _posts/en/newsletters/2021-05-05-newsletter.md:113]:
```suggestion
  core multiprocess] and adds the ability for the `bitcoin-node` process to spawn
```

**Comment by harding** [comment]
Pushed (hopefully) final edits, including updated topics.  Reviewed the excellent contributions by @bitschmidty, @jnewbery, @Xekyo, @dongcarl, and @jonatack.  Thanks everyone!

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+- [BIPs #1104][] updates the [BIP341][] specification of [taproot][topic
+  taproot] with activation parameters based on the Speedy Trial proposal
+  (see [Newsletter #139][news139 speedy trial]).
+
+## Footnotes
+
+[^missing-peer]:
+    The data loss protection protocol, and other proposed methods such
+    as [covert requests for mutual channel closes][news128 covert] requires your channel peer to
```
Comment (jonatack) [inline _posts/en/newsletters/2021-05-05-newsletter.md:163]: - add comma or alternatively, s/requires/require/
- consider s/peer/peers/ as they are referred to as "they" in the next sentence

```suggestion
    as [covert requests for mutual channel closes][news128 covert], requires your channel peers to
```

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+  data. The RewindBlockIndex logic discarded its copies of such blocks,
+  re-downloaded them in their complete form, and validated them using segwit's rules. As pre-segwit
+  nodes have been end-of-life since 2018, the scenario has become uncommon.
+  Future releases will instead prompt the user to reindex for an equivalent
+  outcome.
+
+- [LND #5159][] builds on [previous work][news144 lnd ampsendpayment] to add
+  support for making spontaneous Atomic Multipath Payments (AMPs) by manually
+  specifying payment parameters to the `sendpayment` RPC. Invoking `sendpayment`
+  with an AMP invoice is expected to be implemented in followup PRs.
```
Comment (jonatack) [inline _posts/en/newsletters/2021-05-05-newsletter.md:145]: s/followup/follow-up/

File: _posts/en/newsletters/2021-05-05-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+
+- [Rust-Lightning #893][] only allows accepting a payment if it included
```
Comment (jonatack) [inline _posts/en/newsletters/2021-05-05-newsletter.md:147]: s/included/includes/

**Review by jonatack — COMMENTED** [review]
A final read-through and-click-through. Saw only minor things.

**Comment by bitschmidty** [comment]
Made a couple updates per @jonatack feedback. Squashed and merged.

Pleasure working with this team week in and week out. Thanks to all contributors this week: @jonatack @harding @jnewbery @adamjonas @Xekyo @dongcarl

**Pull request merged**